iree/base/wait_handle_test.cc

```diff
@@ -14,538 +14,803 @@
 
 #include "iree/base/wait_handle.h"
 
-#include <unistd.h>
+#include <thread>
 
-#include <string>
-#include <thread>  // NOLINT
-#include <type_traits>
```

Comment:
ugh GitHub's diff in this file is so bad.. so many arbitrary breaks between old and new code